integration_tests/mockApis/cemo.ts

```diff
@@ -21,7 +22,7 @@ const ping = (httpStatus = 200) =>
     },
   })
 
-export const mockApiOrder = (status = 'IN_PROGRESS') => ({
+export const mockApiOrder = (status: OrderStatus = 'IN_PROGRESS') => ({
```

Comment:
This is using the output models of the parser rather than the input models (e.g. `z.input<typeof OrderModel>`) https://zod.dev/?id=type-inference

Reply:
changed to string